Calculator in C language

chykB opened the pull request:
This pull request adds a simple command-line calculator that uses function pointers. The calculator can perform the basic arithmetic operations of addition, multiplication, division, and subtraction.

MartyOfMCA commented:
Hi there, I'd advice you provide a description for any future pull requests. 

We are trying to equip ourselves with the proper workflow when using GitHub. Also take note of the following important points:

1. Make sure you sync your forked repository with the main repository

      **You can see from the attached photo that this repository is missing 4 commits from the main repository**
![image](https://github.com/ALX-SE-Algorithmia/Demo-Project/assets/104218489/c11d4471-e02f-41b8-be1b-65898d57db71)


3. Make sure you create a directory for your project in the repository you forked. The name for your directory should be your GitHub username like attached photo

![image](https://github.com/ALX-SE-Algorithmia/Demo-Project/assets/104218489/bcef9d24-f0af-4538-91a6-a9da8696d8bc)

4. Lastly, make sure you add the README.md file to the directory you create

chykB commented:
Okay, I will make the needed correction.

Thanks

On Tue, 11 Jul 2023, 16:22 Martin, ***@***.***> wrote:

> Hi there, I'd advice you provide a description for any future pull
> requests.
>
> We are trying to equip ourselves with the proper workflow when using
> GitHub. Also take note of the following important points:
>
>    1.
>
>    Make sure you sync your forked repository with the main repository
>
>    *You can see from the attached photo that this repository is missing 4
>    commits from the main repository*
>    [image: image]
>    <https://user-images.githubusercontent.com/104218489/252710406-c11d4471-e02f-41b8-be1b-65898d57db71.png>
>    2.
>
>    Make sure you create a directory for your project in the repository
>    you forked. The name for your directory should be your GitHub username like
>    attached photo
>
> [image: image]
> <https://user-images.githubusercontent.com/104218489/252711536-bcef9d24-f0af-4538-91a6-a9da8696d8bc.png>
>
>    1. Lastly, make sure you add the README.md file to the directory you
>    create
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ALX-SE-Algorithmia/Demo-Project/pull/23#issuecomment-1631027754>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY2YTSXZ4T64J7HK5MP5G23XPVVR7ANCNFSM6AAAAAA2GD23MQ>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

Jarhodee reviewed (changes requested):
line 23 comment correction: prompt the user to enter two numbers

MartyOfMCA commented:
I noticed you have two source code files in your request. Kindly do well to remove the first `calc.c` file in your root directory. You should have just one directory with all your source code files

![image](https://github.com/ALX-SE-Algorithmia/Demo-Project/assets/104218489/238dc343-d391-49ed-be40-88a460e97fe7)

The first `calc.c` has to be deleted. **And also, make sure the directory you created is named in lowercases**

You can use the command: `mv chykB chykb`

chykB commented:
> I noticed you have two source code files in your request. Kindly do well to remove the first `calc.c` file in your root directory. You should have just one directory with all your source code files
> 
> ![image](https://user-images.githubusercontent.com/104218489/252769774-238dc343-d391-49ed-be40-88a460e97fe7.png)
> 
> The first `calc.c` has to be deleted. **And also, make sure the directory you created is named in lowercases**
> 
> You can use the command: `mv chykB chykb`

I have moved the directory chykB to chykb in my local committed and pushed, I have also removed the first source file. Thanks.

MartyOfMCA commented:
Hello @chykB I can still see the directory name as `chykB` rather than `chykb`. We want to have consistent directory names for every contribution we all make

![image](https://github.com/ALX-SE-Algorithmia/Demo-Project/assets/104218489/43845667-f317-4921-a118-8276ebed5d63)

chykB commented:
> Hello @chykB I can still see the directory name as `chykB` rather than `chykb`. We want to have consistent directory names for every contribution we all make
> 
> ![image](https://user-images.githubusercontent.com/104218489/252801993-43845667-f317-4921-a118-8276ebed5d63.png)

Done
